components/languages.js

```diff
@@ -27,6 +27,7 @@ export const LANGUAGES = {
   tw: ['TraditionalChinese'],
   tr: ['Turkish'],
   ua: ['Ukrainian', 'sans-serif'],
+  ur: ['Urdu'],
```

Comment:
`ur` isn't a valid country identifier, this should probably be `pk` for Pakistan.

Reply:
I've thought about that a little bit before.  Using a country for a language name isn't always great, and can sometimes cause issues.  Not sure what the solution here is though - probably Pakistan, yes, though it is widely used in India.

Reply:
Yeah I had that thought while reviewing this. The main issue is flags as an identifier, but the dropdown _could_ be changed to use language names instead of flags.